### DISCUSSION TITLE
Create a Shiny interactive qmd document within a Quarto website

### DISCUSSION THREAD

**Discussion by jenny-ket** [quarto-dev/quarto-cli#3644] [Q&A]
Hello, 

I've been working on creating a Quarto website, and would like some of the pages to have some basic interactive shiny capability (filtering a map by state) and would like to host the website on shinyapps.io.

 I have encountered an issue exactly as described in the discussion linked below, down to the same error message:
[Discussion 1561](https://github.com/quarto-dev/quarto-cli/discussions/1561)

Because this discussion is several months old, and the response from jjallaire notes that an extension is in the works, I want to re-raise this issue and ask if what I would like to do is possible today, or if there is a work-around for a quarto website hosted on shinyapps.io.

**Comment by mcanouil** [comment]
Still the same answer, but the shinylive Quarto extension was released few weeks ago.
For now, the same thing is not possible for R, but will in the future (see <https://github.com/georgestagg/webR>, note this is a project independent and external to Quarto).

**Reply by jenny-ket** [reply]
Thank you @mcanouil ! 

I was also discussing this with Ian Dinnie at rstudio, and he tested an awesome work-around for people looking to use shinyapps.io for shiny content -- essentially if you create a hosted shiny app on shinyapps.io, you could then iframe the shiny components, referencing the url for the app on shinyapps.io. Noting in case this works for anyone else who needs some simple shiny components.

**Reply by mcanouil** [reply]
Yep, but using iframe is not the same thing.
It's really a workaround.

Depending on the interactivity you are looking for, Observable might be an answer.
